mars/web/api.py

```diff
@@ -178,3 +221,6 @@ def get(self):
 register_web_handler('/api/session/(?P<session_id>[^/]+)/graph/(?P<graph_key>[^/]+)', GraphApiHandler)
 register_web_handler('/api/session/(?P<session_id>[^/]+)/graph/(?P<graph_key>[^/]+)/data/(?P<tileable_key>[^/]+)',
                      GraphDataHandler)
+register_web_handler('/api/session/(?P<session_id>[^/]+)/mutable-tensor', MutableTensorHandler)
+register_web_handler('/api/session/(?P<session_id>[^/]+)/mutable-tensor/write', MutableTensorWriteHandler)
```

Comment:
This API design does not conform to RESTful styles. May change to ``/api/session/(?P<session_id>[^/]+)/mutable-tensor/(?P<name>[^/]+) and use GET / PUT / POST methods to handle data read / write / seal.

Reply:
Will revise.

Reply:
hi @wjsi Mutable tensor requires four endpoints: create/get/write/seal. Is it ok to use the following mapping?

+ `POST` for `create`
+ `HEAD` for `get` (is `HEAD` ok here ?)
+ `PUT` for `write`
+ `GET` for `seal`

Reply:
``GET`` shall not cause any side-effect on the storage. Therefore create and write can be merged into ``PUT`` and the implementation of API decide whether to create while ``POST`` for seal, or ``POST`` with an ``action`` indicating create or seal. If I were the writer of this PR, I would prefer the former solution.

Reply:
> create and write can be merged into PUT and the implementation of API decide whether to create while POST for seal

Nice suggestion, thanks!

Reply:
I have revised the endpoint by use `POST` for both `write` and `seal` (easier to distinguish since `write` has body payload and `seal` doesn't have, and `write`'s body payload is raw bytes, not json, making `create` and `write` are harder to distinguish without extra paramter).

Now all four API of mutable tensor share the same HTTP endpoint.

Reply:
Customized headers can also be used to pass string meta data if you do not like query strings.